### DISCUSSION TITLE
Add support for Jinja2

### DISCUSSION THREAD

**Discussion by erinc** [cloudflare/workerd#1936] [Python Packages]
It would be great to render HTML templates with Starlette , Quart, and Flask.

**Comment by securitypedant** [comment]
Please also see this request, https://github.com/cloudflare/workerd/discussions/1938

**Reply by septatrix** [reply]
As well as #1932

**Comment by Paper-Dragon** [comment]
+1

**Comment by ryanking13** [comment]
jinja2 works in python workers now
